cpu/stm32/include/periph/wl/Kconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2021 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USEMODULE_CPU_FAM_STM32WL_DEBUG
+    bool "STM2WL Debugging"
+    depends on CPU_MODEL_STM32WL55JC
+
+if KCONFIG_USEMODULE_CPU_FAM_STM32WL_DEBUG
```

Comment:
Following the style of the existing Kconfig files for STM32, this `menuconfig` can be avoided, just keep the configuration option.

Reply:
Okay